string status
geometry_msgs/Pose2D position
string robot_name
geometry_msgs/Pose2D trackedGoal